doc/RecordingControl.xml

```diff
@@ -2119,13 +2124,151 @@ secfrac = "." 1*6DIGIT</programlisting>
         </variablelist>
       </para>
       <para>
-        The device shall create an individual initialization vector for each segment.
-        The device shall encrypt with the <literal>Key</literal> configured for the encryption entry as the encryption key.
-        The device shall interpret the <literal>KID</literal> configured for the encryption entry as a 16-byte hexadecimal value and use this value as the key identifier.
+        <literal>Key/KID</literal> and <literal>AsymmetricEncryption</literal> elements of the encryption entry are mutually exclusive.
       </para>
       <para>
+        When using <literal>Key/KID</literal> then the device shall interpret the configured <literal>KID</literal> for the encryption entry as a 16-byte hexadecimal value and use this value as the key identifier.
+        The device shall include a Standard CENC PSSH box according to <xref linkend="_refKeyStorageStandardCenc" />.
+      </para>
+      <para>
+        When using <literal>AsymmetricEncryption</literal> then the device shall generate a <literal>KID/Key</literal> pair and encrypt the <literal>Key</literal> with each certificate defined in the configuration. The encrypted keys shall be stored in the file using the Asymmetric key system PSSH box as defined in <xref linkend="_refKeyStorageKeyRotationSystem" />.
+        When <literal>KeyRotationDuration</literal> is set then the device shall generate a new <literal>KID/Key</literal> pair at the specified time interval, but still use the current key until the segment is finished.
+        New segments shall use the latest generated <literal>Key</literal> for its encryption.
+      </para>
+      <para>
+        The device shall create a unique initialization vector for each segment.
```

Comment:
Shouldn't an initialization vector provided for each fragment to support playback from a fragment boundary?

Reply:
I suggest to require a sample encrypt box 'senc' with IV for each fragment that is listed in the fragement random access box.